content/docs/pulumi-cloud/deployments/api.md

```diff
@@ -380,6 +381,24 @@ The GitHub block describes settings for Pulumi Deployments' GitHub integration.
 }
 ```
 
+### CacheOptions
+
+Cache Options block defines settings related to dependency caching during Deployments. Reading dependencies from cache makes deployments significantly faster. Learn more about dependency caching on the [FAQ page](/docs/pulumi-cloud/deployments/faq/#dependencycaching).
```

Comment:
minor wording things:
The Cache Options block defines settings related to dependency caching during Deployments. Using dependencies from the cache makes deployments significantly faster.

also the url is missing a hyphen: `#dependency-caching`